File: languages/java/src/ConsoleApp.java
public class ConsoleApp {

    public static void main(String[] args) {
        System.out.println("Starting sample application created in Java");

        ServiceInterface service = new DefaultServiceImpl();
        ServiceResult serviceResult = service.executeRequest();

        System.out.printf("Result: %s%n", serviceResult);
    }
}

File: languages/kotlin/src/ConsoleApp.kt
class ConsoleApp {

    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            println("Starting sample application created in Kotlin")

            val service: ServiceInterface = DefaultServiceImpl()
            val serviceResult = service.executeRequest()

            println("Result: $serviceResult")

        }
    }
}